design/synchronizer/auto-failover.md

```diff
@@ -0,0 +1,92 @@
+# Synchronizer Auto-Failover
+
+The following is an overview of the workflow for a Vault Sychronizer to gain leadership:
+
+![](./activity-diagram.png)
+
+Given the following Policy:
+```yml
+- !policy
+  id: conjur
+  body:
+  - !policy
+    id: vault-synchronizer
+    body:
+    - !variable leader-expiration
+
+    # The Group all Synchronizer hosts need to be added to.
+    - !group members
+
+    - !permit
+        role: !group members
+        privileges: [ read, execute, update ]
+        resources:
+      - !variable leader-expiration
+
+      - !policy
+        id: leader
+        owner: !group members
+
+      - !policy
+        id: vault-lines-of-business
+        owner: !group members
+        body:
+        - !variable
+              id: lob-1
+              mime_type: application/json
+        - !variable
+              id: lob-2
+              mime_type: application/json
+```
+
+The following are the steps (using the API) listed above:
+
+1. Update `leader-expiration` to future time:
+
+    ```sh
+    POST /secrets/{account}/variable/conjur%2Fvault-synchronizer%2Fleader-expiration
```

Comment:
"secrets"?
I thought it would be something like "conjur/vault-synchronizer/...."
but I guess this might be my lack of knowledge here

how about creating/updating variables for LOBs - will that also be under "secrets"?

Reply:
In this case `secrets` is the resource type.  `conjur%2Fvault-synchronizer%2Fleader-expiration` is the url escaped variable ID.